drivers/src/fuse_bank.rs

```diff
@@ -223,9 +226,10 @@ impl FuseBank {
     /// # Returns
     ///     runtime security version number
     ///
-    pub fn runtime_svn() -> u64 {
+    pub fn runtime_svn(&self) -> u32 {
         let soc_ifc_regs = soc_ifc::RegisterBlock::soc_ifc_reg();
-        (soc_ifc_regs.fuse_runtime_svn().at(1).read() as u64) << 32
-            | soc_ifc_regs.fuse_runtime_svn().at(0).read() as u64
+        64 - ((soc_ifc_regs.fuse_runtime_svn().at(1).read() as u64) << 32
```

Comment:
For the record, I think this leading-zero  stuff should be in its own function with unit tests to make it easier for folks to review (and trust).

Reply:
Will fix as a separate PR.